core/livepeernode.go

```diff
@@ -261,6 +261,19 @@ func (n *LivepeerNode) GetBasePriceForCap(b_eth_addr string, cap Capability, mod
 	return nil
 }
 
+func (n *LivepeerNode) GetCapsPrices(b_eth_addr string) *CapabilityPrices {
```

Comment:
Is this change related to this PR? I cannot connect the dots why we change anything about prices here.

Reply:
This is added to enable getting prices for all capabilities to be able to send all prices in the /getNetworkCapabilities endpoint on the local Gateway webserver.